dwml-data-parsers/parameter.js

```diff
@@ -72,9 +72,23 @@ var parameterParser = {
 
     while (i < values.length) {
       var currentValue = values[i];
+
+      currentTimeFrame = timeFrames[timeFrameCounter];
       if (currentValue.name === 'value') {
         currentTimeFrame = timeFrames[timeFrameCounter];
-        results.push(_.extend({}, currentTimeFrame, {value: currentValue.content }));
+        results.push(_.extend({}, currentTimeFrame, {value: currentValue.content }));  
+        timeFrameCounter++;
+      }
+      if (currentValue.name === 'weather-conditions') {
```

Comment:
if this line were

```
if (currentValue.name === 'weather-conditions' && currentValue.children && currentValue.children.length) {
```

I think you could remove the subsequent checks below.  But that might exclude the summary.  Thoughts? 

If that doesn't work, maybe you could create a `hasChildren` variable where you check for both the existence of the children, and the length:

```
var hasChildren = currentValue.children && currentValue.children.length;
```

I think that's a bit safer, and might drop a few characters below